plugins/woocommerce-blocks/tests/e2e/tests/product-collection/collections/collections.block_theme.spec.ts

```diff
@@ -0,0 +1,221 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect } from '@woocommerce/e2e-utils';
+
+/**
+ * Internal dependencies
+ */
+import ProductCollectionPage, { SELECTORS } from '../product-collection.page';
+
+const test = base.extend< { pageObject: ProductCollectionPage } >( {
+	pageObject: async ( { page, admin, editor }, use ) => {
+		const pageObject = new ProductCollectionPage( {
+			page,
+			admin,
+			editor,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( 'Product Collection', () => {
+	test.describe( 'Collections', () => {
```

Comment:
Sorry for the post-merge comment, but it seems unnecessarily nested here. We should keep the structure as flat as possible, for example:

```js
test.describe( 'Product Collection: Collections' () => { // ...
```